lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -245,11 +245,13 @@ def add_payment_sender_data(post, options)
 
         sender = options[:sender]
 
+        birthdate = sender[:dob] || sender[:date_of_birth]
+
         post[:sender] = {}
         post[:sender][:type] = sender[:type] if sender[:type]
         post[:sender][:first_name] = sender[:first_name] if sender[:first_name]
         post[:sender][:last_name] = sender[:last_name] if sender[:last_name]
-        post[:sender][:dob] = sender[:dob] if sender[:dob]
+        post[:sender][:date_of_birth] = birthdate
```

Comment:
```suggestion
        post[:sender][:date_of_birth] = birthdate if birthdate
```

In case the API is going to complain about a blank/empty value being sent, I think we should guard with `if` like the others